docs/community/documentation-guidelines.md

```diff
@@ -4,23 +4,23 @@ title: Documentation guidelines
 ---
 # Documentation guidelines
 
-When developers or users have problems with software the usual forum quip is to read the manual. Sometimes in nice tones and others not so nice. It's great when the documentation is succinct and easy to read and comprehend. All to often though there are huge swathes of missing, incomplete or downright wrong bits that leave people more confused than before they read the documentation.
+When developers or users have problems with software the usual forum quip is to read the manual. Sometimes in nice tones and others not so nice. It's great when the documentation is succinct and easy to read and comprehend. All too often, though, there are huge swathes of missing, incomplete, or downright wrong bits that leave people more confused than they were before they read the documentation.
 
-So the call goes out for people to help build up the documentation. Which is great until you have a lot of people with their own ideas how everything should be laid out trying to contribute. To alleviate the confusion guidelines are setup. This document illustrates the documentation guidelines for this project.
+So the call goes out for people to help build up the documentation. Which is great until you have a lot of people with their own ideas how everything should be laid out trying to contribute. To alleviate the confusion, guidelines are setup. This document illustrates the documentation guidelines for this project.
 
-There is a ton of work that still needs to be done especially in the API documentation department. Please don't hesitate to join and contribute to the project. We welcome everyone and could use your help.
+There is a ton of work that still needs to be done, especially in the API documentation department. Please don't hesitate to join and contribute to the project. We welcome everyone and could use your help.
 
 ## Website
 
-When writing documentation for the website, the project uses [Markdown](http://daringfireball.net/projects/markdown/syntax) when crafting the documents. The rendering of the website is done with Marked.JS. Thus, any editor based on this will give you the best preview/edit experience, such as [Atom](https://atom.io/) or [StackEdit](https://stackedit.io/).
+This the project uses [Markdown](http://daringfireball.net/projects/markdown/syntax) for crafting the documents for the website. The rendering of the website is done with Marked.JS. Thus, any editor based on this will give you the best preview/edit experience, such as [Atom](https://atom.io/) or [StackEdit](https://stackedit.io/).
```

Comment:
"This the project" should either be "This project" or "The project".

Other than this change, it looks good to me.